contracts/MessageBox.sol

```diff
@@ -11,10 +11,13 @@ contract MessageBox {
     }
 
     function message() external view returns (string memory) {
-        // TODO: Signed queries not supported yet. https://github.com/oasisprotocol/sapphire-paratime/issues/347
-        // if (msg.sender != author) {
-        // revert("not allowed");
-        // }
+      return _message;
+    }
+
+    function message_signed() external view returns (string memory) {
```

Comment:
Let's require signed version only, since this is the whole idea of the wrapper. Also remove the helpers from .py files.

```suggestion
```

Reply:
Agreed, removed message_signed and reduntant code.